Plugins/GitUIPluginInterfaces/GitUIEventArgs.cs

```diff
@@ -10,12 +11,17 @@ public GitUIEventArgs(IWin32Window? ownerForm, IGitUICommands gitUICommands)
         {
             OwnerForm = ownerForm;
             GitUICommands = gitUICommands;
+            _refs = new(GitModule);
         }
 
         public IGitUICommands GitUICommands { get; }
 
         public IWin32Window? OwnerForm { get; }
 
         public IGitModule GitModule => GitUICommands.GitModule;
+
+        private readonly LazyGitRefsProvider _refs;
+
+        public IReadOnlyList<IGitRef> GetRefs(RefsFilter filter) => _refs.GetRefs(filter);
```

Comment:
Mixing `private` and `public`?

Reply:
What do you want to change?

Reply:
`private` and `protected` properties should be in a separate section, i.e. not intermixed with `public`.